android/java/org/chromium/chrome/browser/crypto_wallet/fragments/onboarding_fragments/DownloadComponentProgressFragment.java

```diff
@@ -0,0 +1,115 @@
+/* Copyright (c) 2023 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+package org.chromium.chrome.browser.crypto_wallet.fragments.onboarding_fragments;
+
+import android.os.Bundle;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.TextView;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.fragment.app.Fragment;
+
+import org.chromium.base.TimeUtils.ElapsedRealtimeMillisTimer;
+import org.chromium.chrome.R;
+import org.chromium.chrome.browser.component_updater.BraveComponentUpdater;
+import org.chromium.chrome.browser.crypto_wallet.util.WalletDataFilesInstaller;
+
+/**
+ * A fragment to display progress of Brave Wallet data files component
+ * download. Used on all Brave Wallet Onboarding fragments.
+ */
+public class DownloadComponentProgressFragment extends Fragment {
+    private static final String TAG = "DCPF";
+    private static final String WALLET_COMPONENT_ID =
+            WalletDataFilesInstaller.getWalletDataFilesComponentId();
+    private TextView mComponentDownloadProgress;
+    private BraveComponentUpdater.ComponentUpdaterListener mComponentUpdaterListener;
+    private ElapsedRealtimeMillisTimer mGracePeriodNoDisplayTimer;
+    private static final long GRACE_NO_DISPLAY_MSEC = 1000;
+
+    public DownloadComponentProgressFragment() {}
```

Comment:
Empty constructor can be removed.

Reply:
yes, fixed at https://github.com/brave/brave-core/pull/19086/commits/54818d8dcc99e3f267ebe48dde6da4ba6625545f#diff-cc80fe4b95c53c61590521b7c292b734ca565052dda88afa2c5b252ae3889572L36